src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala

```diff
@@ -178,6 +181,21 @@ class EasyConvertBagToDepositApp(configuration: Configuration) extends DebugEnha
     }
   }
 
+  private def checkAgreementsXml(depositorId: String, agreementsFile: File) = {
+    if( ! agreementsFile.exists ) {
+      val templateFile: File =  configuration.agreementsPath / (depositorId+"-agreements.xml")
```

Comment:
I suppose `easy-validate-dans-bag` needs to be adjusted too, to be more tolerant on this file name.

Reply:
no, the file is renamed 'agreements.xml' in the copy

Reply:
I was blind as a bat.